addons/addon-base-raas-ui/packages/base-raas-ui/src/models/studies/Study.js

```diff
@@ -15,10 +15,44 @@
 
 import { types, applySnapshot } from 'mobx-state-tree';
 
+import _ from 'lodash';
 import { StudyFilesStore } from './StudyFilesStore';
 import { StudyPermissionsStore } from './StudyPermissionsStore';
 import { categories } from './categories';
 
+// 'pending', 'error', 'default'
+// Note: When study is available, the status column is empty
```

Comment:
Is that true?  That is only in the database, but when we convert the dbEntity to a studyEntity in the backend we populate the status as 'reachable' if there is no status in the database.  Can you confirm? thanks